torch/distributions/__init__.py

```diff
@@ -39,6 +39,8 @@
 from .normal import Normal
 from .exponential import Exponential
 from .laplace import Laplace
+from .chi2 import Chi2
```

Comment:
@alicanb Would you mind importing `Uniform` in this PR to help clean up?
```py
from .uniform import Uniform
```
since it was missed in #4328 ?